docs/get-started/cli-commands.mdx

```diff
@@ -494,66 +499,84 @@ Set block production time in seconds. Default: `2`
 
 ---
 
-### dev flags
+<h4><i>restore</i></h4>
 
-<h4><i>log-level</i></h4>
+<Tabs>
+  <TabItem value="syntax" label="Syntax" default>
+
+    server [--restore RESTORE]
+
+  </TabItem>
+  <TabItem value="example" label="Example">
+
+    server --restore backup.dat
+
+  </TabItem>
+</Tabs>
+
+Restore blocks from the specified archive file
 
+---
+
+<h4><i>access-control-allow-origins</i></h4>
 <Tabs>
   <TabItem value="syntax" label="Syntax" default>
 
-    dev [--log-level LOG_LEVEL]
+    server [--access-control-allow-origins ACCESS_CONTROL_ALLOW_ORIGINS]
 
   </TabItem>
   <TabItem value="example" label="Example">
 
-    dev --log-level DEBUG
+    server --access-control-allow-origins "https://edge-docs.polygon.technology"
 
   </TabItem>
 </Tabs>
 
-Sets the log level for console output. Default: `INFO`.
+Sets the authorized domains to be able to share responses from JSON-RPC requests.   
+Add multiple flags `--access-control-allow-origins "https://example1.com" --access-control-allow-origins "https://example2.com"` to authorize multiple domains.   
+If omitted Access-Control-Allow-Origins header will be set to `*` and all domains will be authorized.
 
 ---
 
-<h4><i>premine</i></h4>
+### dev flags
+
+<h4><i>log-level</i></h4>
 
 <Tabs>
   <TabItem value="syntax" label="Syntax" default>
 
-    dev [--premine ADDRESS:VALUE]
+    dev [--log-level LOG_LEVEL]
 
   </TabItem>
   <TabItem value="example" label="Example">
 
-    dev --premine 0x3956E90e632AEbBF34DEB49b71c28A83Bc029862:1000000000000000000000
+    dev --log-level DEBUG
 
   </TabItem>
 </Tabs>
 
-Sets the premined accounts and balances in the format `address:amount`.
-The amount can be in either decimal or hex.
-Default premined balance: `0x3635C9ADC5DEA00000`.
+Sets the log level for console output. Default: `INFO`.
 
 ---
 
-<h4><i>block-gas-target</i></h4>
+<h4><i>premine</i></h4>
 
 <Tabs>
   <TabItem value="syntax" label="Syntax" default>
 
-    server [--block-gas-target BLOCK_GAS_TARGET]
+    dev [--premine ADDRESS:VALUE]
 
   </TabItem>
   <TabItem value="example" label="Example">
 
-    server --block-gas-target 10000000
+    dev --premine 0x3956E90e632AEbBF34DEB49b71c28A83Bc029862:1000000000000000000000
 
   </TabItem>
 </Tabs>
 
-Sets the target block gas limit for the chain. Default: `8000000`.
-
-A more detailed explanation on the block gas target can be found in the [TxPool section](/docs/architecture/modules/txpool#block-gas-target).
+Sets the premined accounts and balances in the format `address:amount`.
+The amount can be in either decimal or hex.
+Default premined balance: `0x3635C9ADC5DEA00000`.
```

Comment:
I would like to note that this is the value in WEI -> ETH proportion, so for 1000 you need 1000000000000000000000. What do think?

Reply:
Added in bd790b88243930ab6def0740d729a07387349e20